model_compression_toolkit/pytorch/mixed_precision/mixed_precision_wrapper.py

```diff
@@ -68,7 +91,16 @@ def forward(self, x: Any) -> Any:
         Returns:
             torch Tensor which is the output of the wrapped layer on the given input.
         """
-        return self.layer(x)
+        outputs = self.layer(x)
+
+        if self.enable_activation_quantization:
+            # add fake quant to quantize activations with the active number of bits
+            if isinstance(outputs, list):
+                # we assume here that it can't be multiple outputs out of a quantized layer
+                outputs = torch.cat(outputs, dim=0)
```

Comment:
Let's raise an exception in case of multiple outputs and enable activation quantization